atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransactionManagerImpl.java

```diff
@@ -50,7 +50,8 @@
 import com.palantir.lock.v2.TimelockService;
 import com.palantir.timestamp.TimestampService;
 
-/* package */ class SnapshotTransactionManager extends AbstractLockAwareTransactionManager {
+class SnapshotTransactionManagerImpl extends AbstractTransactionManager implements
```

Comment:
We should remove this and just have `SerializableTransactionManagerImpl`, but keep the separation of `SerializableTransaction` vs. `SnapshotTransaction` for now.

It's pretty weird to have SerializableImpl extends SnapshotImpl implements Serializable